minorminer/utils/raster_embedding.py

```diff
@@ -11,13 +11,86 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import warnings
 
-import networkx as nx
 import dwave_networkx as dnx
+import matplotlib.pyplot as plt
+import networkx as nx
 import numpy as np
-import warnings
+
 from minorminer.subgraph import find_subgraph
 
+def visualize_embeddings(H, embeddings=None, title=None, **kwargs):
+    """Visualizes the embeddings using dwave_networkx's layout functions.
+
+    Args:
+        H (networkx.Graph): The input graph to be visualized. If the graph 
+            represents a specialized topology, it must be constructed using 
+            dwave_networkx (e.g., chimera, pegasus, or zephyr graphs).
+        embeddings (list of dict, optional): A list of embeddings where each 
+            embedding is a dictionary mapping nodes of the source graph to 
+            nodes in the target graph. If not provided, only the graph `H` 
+            will be visualized without specific embeddings.
+        title (str, optional): Title of the plot. Defaults to None.
+        **kwargs: Additional keyword arguments passed to the drawing functions 
+            (e.g., node size, font size).
+    Draws:
+        - Specialized layouts: Uses dwave_networkx's `draw_chimera`, 
+          `draw_pegasus`, or `draw_zephyr` if the graph family is identified.
+        - General layouts: Falls back to networkx's `draw_networkx` for 
+          graphs with unknown topology.
+    """
+    fig = plt.gcf() 
+    ax = plt.gca()
+    fig.set_size_inches(10, 8)
```

Comment:
```suggestion
```

Reply:
I suggest we remove this. Rationale is that the function should only do what's necessary, i.e., draw. (also pattern-matching other plotting functions)